spec.md

```diff
@@ -477,7 +477,112 @@ class ClassName
 }
 ```
 
-### 4.4 Methods and Functions
+### 4.4 Property Hooks
```

Comment:
IMO we should add this to the end of 4 rather than displacing everything else in 4. Ideally one could refer to `PER-CS 4.5` and expect it to remain the same across _at least_ minor versions.

Reply:
Agree.